op_builder/hpu/fused_adam.py

```diff
@@ -4,10 +4,85 @@
 
 # DeepSpeed Team
 
-from .builder import CPUOpBuilder
+try:
```

Comment:
@BacharL, is it possible to have a separate PR for this, since it is unrelated to CPUAdam? Splitting will greatly help future searches and references to the respective topics.

Reply:
yes, sorry, it went in in error. I will revert
https://github.com/microsoft/DeepSpeed/pull/5500